<launch>
    <arg name="in_path" default="/home/ildar/catkin_ws/src/socket_streamer/launch/seq1_restamp" />
    <remap from="/image_raw" to="/cam0/image_raw" />
    <remap from="/imu" to="/imu0" />
    
    <!-- <remap from="/vicon/cf1/cf1" to="/vicon/firefly_sbx/firefly_sbx" /> -->

    <node pkg="rosbag" type="play" name="player1" output="screen" 
        args="$(arg in_path).bag 
        --topics /image_raw /imu
        --clock"
        />
</launch>